pkg/kubelet/dockertools/docker.go

```diff
@@ -144,23 +145,32 @@ func filterHTTPError(err error, image string) error {
 }
 
 func (p dockerPuller) Pull(image string, secrets []api.Secret) error {
-	// If no tag was specified, use the default "latest".
-	imageID, tag := parsers.ParseImageName(image)
+	if parsers.GetImageTag(image) == "" {
+		// If no tag is specified, apply the default tag.
```

Comment:
Because from `ParseImageName` we don't know whether the `image` is "name:latest" or just "name", for both cases it will just return "image + latest".